src/docs/transaction-clustering/index.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Sanitizing URL Transactions # TODO: decide on name.
```

Comment:
This might be a good time to pick a definitive name for this feature. Plenty of choices:

* Transaction (name) Normalization
* Sanitizing Transaction Names
* Clustering (URL) Transaction (Names)
* Transaction Grouping

Please leave a vote :)

Reply:
Decided to go for "Clustering URL Transactions" after talking to @jan-auer.